### PR TITLE
[SIG-1998] fixed validation of name in role form

### PR DESCRIPTION
**Pull request by jasperswart**
in this PR:
- validation error on name field only triggers after form submit

### CODE REVIEW

File: src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const StyledInput = styled(Input)`
 
 export const RoleForm = ({ role, permissions, onPatchRole, onSaveRole }) => {
   const [name, setName] = useState('');
+  const [isSubmitting, setSubmitting] = useState(false);
```
Comment (janjaap) [inline src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js]: Why not use the `useFormValidation` hook?

Comment (jasperswart) [reply src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js]: hoe? ik zie het niet in de instellingen branch: wel de definitie, maar niet de toepassing in het userform?

Comment (janjaap) [reply src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js]: You can find the hook in `src/hooks/useFormValidation`. See #494 for implementation details.

Comment (jasperswart) [reply src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js]: can you help me with that on Monday?

Comment (jasperswart) [reply src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js]: gedaan

**Pull request merged**